src/administration/Administration.Service/Controllers/PartnerNetworkController.cs

```diff
@@ -56,6 +57,6 @@ public PartnerNetworkController(IPartnerNetworkBusinessLogic logic)
     [Route("memberCompanies")]
     [ProducesResponseType(StatusCodes.Status200OK)]
     [PublicUrl(CompanyRoleId.ACTIVE_PARTICIPANT, CompanyRoleId.SERVICE_PROVIDER, CompanyRoleId.APP_PROVIDER)]
-    public IAsyncEnumerable<string?> GetAllMemberCompaniesBPNAsync() =>
-        _logic.GetAllMemberCompaniesBPNAsync();
+    public IAsyncEnumerable<string?> GetAllMemberCompaniesBPNAsync([FromBody] IEnumerable<string>? bpnIds) =>
```

Comment:
FromBody isn't possible for Get requests. Please change to FromQuery

```suggestion
    public IAsyncEnumerable<string?> GetAllMemberCompaniesBPNAsync([FromQuery] IEnumerable<string>? bpnIds) =>
```